.gitlab-ci.yml

```diff
@@ -252,6 +252,7 @@ variables:
   SSH_PUBLIC_KEY_RSA: ci.datadog-agent.ssh_public_key_rsa  # agent-devx-loops
   VCPKG_BLOB_SAS_URL: ci.datadog-agent-buildimages.vcpkg_blob_sas_url  # windows-agent
   WINGET_PAT: ci.datadog-agent.winget_pat  # windows-agent
+  # Keep an empty line after this comment
```

Comment:
WDYT about using a comment that could look like `END OF PARAM SECTION` instead of relying on an empty line? I feel like we're more likely to not delete a comment than an empty line

Reply:
Yeah I think the risk of deletion of the empty line is low. But the risk of the addition of a new empty line in the middle of the param section is more likely